products/resourcemanager/terraform.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2018 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Provider::ResourceOverrides
+  Project: !ruby/object:Provider::Terraform::ResourceOverride
+    exclude: true
+  Lien: !ruby/object:Provider::Terraform::ResourceOverride
+    exclude: false
+    examples: |
+      ```hcl
+      resource "random_id" "r" {
+        byte_length = 8
+      }
+
+      resource "google_project" "project" {
+        project_id = "project-${random_id.r.hex}"
+        name = "A very important project!"
+      }
+
+      resource "google_resourcemanager_lien" "lien" {
+        parent = "projects/${google_project.project.number}"
+        restrictions = ["resourcemanager.projects.delete"]
+        origin = "machine-readable-explanation"
+        reason = "This project is very important to me!"
+      }
+      ```
+
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      post_create: templates/terraform/lien_post_create.erb
```

Comment:
For my other PRs using custom code, I have been following a pattern where I put the files in a directory inside templates based on the custom_code injection point. 
https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/custom_flatten
https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/constants
https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/update_encoder